yarn.lock

```diff
@@ -2134,11 +2134,11 @@ ecc-jsbn@~0.1.1:
 
 "eclipse-che-operator@git://github.com/eclipse/che-operator#master":
   version "0.0.0"
-  resolved "git://github.com/eclipse/che-operator#40b3904d69f9c588d67d37d61fe92120a68f8c3a"
+  resolved "git://github.com/eclipse/che-operator#0d51ea2f0d9cbe8bfcc5fa62a09225a280f8875e"
```

Comment:
Revert, please

Reply:
There is no pending PR anymore, we can update it.